### PR TITLE
NS-865 Include basic_attributes table in non-advisee queries

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/NS-865

### CODE REVIEW

File: nessie/lib/queries.py [hunk 1]
```diff
@@ -306,24 +306,30 @@ def get_fetched_non_advisees():
...
-              LEFT JOIN {asc_schema()}.students ascs ON ascs.sid = en.sis_id
-              LEFT JOIN {coe_schema()}.students coe ON coe.sid = en.sis_id
-              LEFT JOIN {undergrads_schema()}.students ug ON ug.sid = en.sis_id
-              LEFT JOIN {student_schema()}.sis_api_profiles_hist_enr hist ON hist.sid = en.sis_id
+    sql = f"""SELECT DISTINCT(attrs.sis_id) AS sid
+              FROM (
+                SELECT sis_id FROM {sis_schema()}.enrollments
+                UNION SELECT sid AS sis_id FROM {sis_schema()}.basic_attributes
+              ) attrs
+              JOIN {asc_schema()}.students ascs ON ascs.sid = attrs.sis_id
```
Comment (lyttam) [inline nessie/lib/queries.py:314]: @paulkerschen Was the switch from a `LEFT JOIN` to a `JOIN` intentional?

Comment (paulkerschen) [reply nessie/lib/queries.py:314]: No! Good eye, thanks.

Comment (paulkerschen) [reply nessie/lib/queries.py]: Amended.

**Comment by lyttam** [comment]
Thanks!

**Pull request merged**